iOS/Shimmer/source/Shimmer/ApiDefinition.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using CoreAnimation;
+using Foundation;
+using ObjCRuntime;
+using UIKit;
+
+namespace Shimmer
+{
+    // @protocol FBShimmering <NSObject>
+    [Protocol, Model]
+    [BaseType(typeof(NSObject), Name = "FBShimmering")]
+    interface Shimmering
+    {
+        // @required @property (getter = isShimmering, assign, readwrite, nonatomic) BOOL shimmering;
+        [Abstract]
+        [Export("shimmering")]
+        bool IsShimmering { [Bind("isShimmering")] get; set; }
```

Comment:
This name is ok but I suggest changing this property to just `Shimmering` due it has a setter property. The `Is` prefix is recommended when is a read-only property. But I leave it to Adlair's decision.